Sources/MacroToolkit/Type.swift

```diff
@@ -174,6 +169,199 @@ public enum `Type`: TypeProtocol, SyntaxExpressibleByStringInterpolation {
     }
 
     // TODO: Implement rest of conversions
+    
+    public func normalized() -> NormalizedType {
+        switch self {
+        case .array(let type):
+            var arrayTypeSyntax: ArrayTypeSyntax = type._baseSyntax
+            var attributedTypeSyntax: AttributedTypeSyntax? = type._attributedSyntax
+            let normalizedElement = Type(arrayTypeSyntax.element).normalized()
+            arrayTypeSyntax.element = TypeSyntax(normalizedElement._syntax)
+            attributedTypeSyntax?.baseType = TypeSyntax(arrayTypeSyntax)
+            
+            var base = "Array<\(arrayTypeSyntax.element)>"
+            if let attributedTypeSyntax {
+                base = base.addingAttributes(from: attributedTypeSyntax)
+            }
+            return NormalizedType(stringLiteral: base)
+        case .classRestriction(let type):
+            // Not handling `_attributedSyntax` because `classRestriction` cannot have any attribute
+            
+            // let normalizedType: NormalizedType = "AnyObject"
```

Comment:
Remove this commented out line and then the blank line above it too

Reply:
Fixed in 71aff30.